io.gitlab.liferooter.TextPieces.gnome.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2023 Gleb Smirnov <glebsmirnov0708@gmail.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+id: io.gitlab.liferooter.TextPieces.gnome
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+
+command: textpieces-gnome
+
+finish-args:
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
+- --env=RUST_LOG=textpieces_gnome=debug
+- --env=G_MESSAGES_DEBUG=none
+- --env=RUST_BACKTRACE=full
+- --talk-name=org.freedesktop.Flatpak
+- --filesystem=~/.local/share/textpieces
+- --filesystem=~/.var/app/com.github.liferooter.textpieces
```

Comment:
these permissions are excessive and shouldn't be.

Reply:
the eol rebase takes care of the migration too.

Reply:
These permissions, as I've said, are required for smooth transition between the old version of Text Pieces (with different ID) and the new one. Currently the application automatically imports the data from the old location. Probably in future it will be remove as excessive.

Reply:
as I said:

> the eol rebase takes care of the migration too.

Reply:
Ok, so I've removed `--filesystem=~/.var/app/com.github.liferooter.textpieces`. But I still can't remove other one, because it's outside the sandbox. It was used not just to make life harder and to break XDG standards, but because of an old bug of XDG Desktop Portals that didn't allow applications to get read-write access to each other's files, so it was impossible to open scripts inside `~/.var/app/com.github.liferooter.textpieces/data/textpieces` with an editor. So I've used this weird hack to bypass it. Nowadays the bug is fixed, but old version used this directory as a data storage, so it's required for migration to work.

Yeah, I know, it's ugly. But as is.